test/unit/core/simd/detail/builtin_simd_test.cpp

```diff
@@ -141,8 +190,19 @@ TEST(builtin_simd, simd_concept)
 {
     EXPECT_FALSE(simd_concept<short>);
     EXPECT_FALSE(simd_concept<int>);
-    EXPECT_FALSE(simd_concept<int[15]>);
-    EXPECT_FALSE(simd_concept<int*>);
+    EXPECT_FALSE(simd_concept<incomplete::type>);
+    EXPECT_FALSE(simd_concept<incomplete::template_type<int>>);
```

Comment:
so without the explicit protection with the !is_pointer gcc will hard fail on the requires clause? Have you reported this as a bug to gcc? is there any update?

Reply:
yes, it has no SFINAE behaviour then.